setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = ads2inspire
+version = 0.1.0
```

Comment:
Should I be using semantic or calendar versioning?
Should the source have access to the version string?

Reply:
> Should I be using semantic or calendar versioning?

This is a debate that rages. I use semantic versioning for all my projects, but I have my own biases. I can send you videos that argue for calendar versioning if you want to hear some other agruments.

> Should the source have access to the version string?

We do this for `pyhf`, but this can be added in another PR. Though if you'd prefer to have that in now that's not a problem.